addons/website/static/src/scss/website.scss

```diff
@@ -243,6 +243,24 @@ body:not(.o_fullscreen) {
         @extend .navbar-dark;
     }
     @include o-add-gradient('menu-gradient');
+    // Reset dropdown and search input.
+    .dropdown-menu .dropdown-item {
+        @extend .dropdown-item;
+    }
+    .dropdown-menu .dropdown-item-text.text-muted {
+        color: $text-muted !important;
+
+        a.s_searchbar_fuzzy_submit {
+            color: theme-color('primary');
+
+            @include hover-focus {
+                color: $navbar-light-active-color;
+            }
+        }
+    }
+    .o_searchbar_form .btn-primary {
+        border-color: $input-border-color;
+    }
```

Comment:
I think this whole commit should be about fixing the `o-apply-colors` mixin:
```diff
 @if type-of($-related-color) == 'number' {
     // This is a preset to be applied, just extend it. This should probably
     // be avoided and use the class in XML if possible.
+    @extend .o_cc;
     @extend .o_cc#{$-related-color};
```

Could you check if I am right?

Reply:
It works for most of it.
I however kept the
```js
    .o_searchbar_form .btn-primary {
        border-color: $input-border-color;
    }
```
but I put it inside the dark-specific `if`.

Reply:
Ultimately I also needed to keep the text-muted part. I moved them inside the o_ccN definition, and applied the button border change to any button nested inside an `input-group-append`.